### PR TITLE
Tracker ui improvements

### PR DESCRIPTION
**Pull request by erynnb**
- Improve UI for processing and reading donation comments
- Add ability to mark a donation comment as "priority"
- Add ability to supply a reason for approving a donation comment

### CODE REVIEW

**Comment by Lange** [comment]
I tagged both @bcongdon and @justinkim as potential reviewers. Don't need both to review, just one is okay, whichever has time first.

File: models/donation.py [hunk 1]
```diff
@@ -49,6 +49,7 @@ class Donation(models.Model):
   bidstate = models.CharField(max_length=255,default='PENDING',choices=(('PENDING', 'Pending'), ('IGNORED', 'Ignored'), ('PROCESSED', 'Processed'), ('FLAGGED', 'Flagged')),verbose_name='Bid State')
   readstate = models.CharField(max_length=255,default='PENDING',choices=(('PENDING', 'Pending'), ('READY', 'Ready to Read'), ('IGNORED', 'Ignored'), ('READ', 'Read'), ('FLAGGED', 'Flagged')),verbose_name='Read State')
   commentstate = models.CharField(max_length=255,default='ABSENT',choices=(('ABSENT', 'Absent'), ('PENDING', 'Pending'), ('DENIED', 'Denied'), ('APPROVED', 'Approved'), ('FLAGGED', 'Flagged')),verbose_name='Comment State')
+  prioritystate = models.BooleanField(default=False)
```
Comment (yuna9) [inline models/donation.py:52]: If the data type is a boolean, a name that could communicate intention better would be `is_priority` or similar.

File: templates/admin/process_donations.html [hunk 2]
```diff
@@ -58,23 +71,48 @@
 
   resultsTable = $("#id_result_set");
   setTimeout(runSearch, 0);
+
```
Comment (yuna9) [inline templates/admin/process_donations.html:74]: rando newlines

**Review by yuna9 — APPROVED** [review]
lgtm, just needs manual testing probably

**Comment by Lange** [comment]
ok, I rebased this on master so that we have accurate tests now.

it looks like this is failing because it needs a migration to add the new database field

**Comment by Lange** [comment]
@erynnb here's documentation on how to make a new migration: https://docs.djangoproject.com/en/1.8/topics/migrations/#adding-migrations-to-apps

**Comment by yuna9** [comment]
UI reviewed by Whisker, no additional asks.

**Pull request merged**